### ISSUE TITLE
Reference predicted result on exemplar images

### ISSUE THREAD

**Issue by Masaaki-75** [mlmed/torchxrayvision#160]
Hi, thanks for the great work!

I was wondering if there is a reference predicted result on the given exemplar images (like https://github.com/mlmed/torchxrayvision/blob/master/tests/00000001_000.png), so other users can quickly check if they implement the model right (they need to customize their inference code in some cases)?

For example, for the image `00000001_000.png` and the model `"densenet121-res224-all"`, i got the following result (without a `xrv.datasets.XRayResizer(224)` transform):
```python
{
'Atelectasis': 0.50504273, 
'Consolidation': 0.3058891, 
'Infiltration': 0.51544267, 
'Pneumothorax': 0.24868299, 
'Edema': 0.27407962, 
'Emphysema': 0.5036348, 
'Fibrosis': 0.53295517, 
'Effusion': 0.4028573, 
'Pneumonia': 0.30947196, 
'Pleural_Thickening': 0.51026535, 
'Cardiomegaly': 0.66010153, 
'Nodule': 0.5091888, 
'Mass': 0.32262152, 
'Hernia': 0.011928638, 
'Lung Lesion': 0.22214857, 
'Fracture': 0.32384807, 
'Lung Opacity': 0.27742577, 
'Enlarged Cardiomediastinum': 0.4100802}
```

But i have no idea whether or not this predicted result is as expected. So a reference result will be useful for checking.

Another question is that, when training models on the MIMIC-CXR dataset, did you use the labels provided by MIMIC-CXR-JPG (from `mimic-cxr-2.0.0-negbio.csv.gz` or `mimic-cxr-2.0.0-chexpert.csv.gz`)?

Thank you!

**Comment by ieee8023** [comment]
> I was wondering if there is a reference predicted result on the given exemplar images

There are some sample outputs here using the 'densenet121-res224-all' weights: https://github.com/mlmed/torchxrayvision?tab=readme-ov-file#getting-started for the files 00000001_000.png and 16747_3_1.jpg.

Let me know if this isn't enough to verify that your code is working.

Also, there was some prediction inconsistency when automatic resizing was happening because nn.Upsample was being used and it resized the images differently to how skimage was resizing them. This is fixed now if you update to version 1.3.2. Not sure if this could be related to the issues you were having.

> Another question is that, when training models on the MIMIC-CXR dataset, did you use the labels provided by MIMIC-CXR-JPG (from mimic-cxr-2.0.0-negbio.csv.gz or mimic-cxr-2.0.0-chexpert.csv.gz)?

There are two sets of weights, one using each of the labels. They are mimic_nb (negbio) and mimic_ch (chexpert).

https://github.com/mlmed/torchxrayvision/blob/57916b36878cb2f6e7e0f3a852f6c6ff1ae07b21/torchxrayvision/models.py#L66

**Comment by Masaaki-75** [comment]
Thank you for the helpful clarification! I will check the sample output of those sample images :)

> There are two sets of weights, one using each of the labels. They are mimic_nb (negbio) and mimic_ch (chexpert).

What about the 'densenet121-res224-all' model? Its training set should involve MIMIC-CXR dataset.

**Comment by ieee8023** [comment]
The 'all' model is described in more detail here: https://arxiv.org/abs/2002.02497

It is trained on these datasets: nih, pc, chex, mimic_ch, google, openi, rsna

**Comment by Masaaki-75** [comment]
Appreciate the info!
